spec/core_functions/math/log.hrx

```diff
@@ -0,0 +1,162 @@
+<===> options.yml
+---
+:todo:
+- sass/libsass#3044
+
+<===>
+================================================================================
+<===> input.scss
+@use "sass:math" as math;
+a {b: math.log(2)}
+
+<===> output.css
+a {
+  b: 0.6931471806;
+}
+
+<===>
+================================================================================
+<===> negative/input.scss
+@use "sass:math" as math;
+a {b: math.log(-1)}
+
+<===> negative/output.css
+a {
+  b: NaN;
+}
+
+<===>
+================================================================================
+<===> zero/input.scss
+@use "sass:math" as math;
+a {b: math.log(0)}
+
+<===> zero/output.css
+a {
+  b: -Infinity;
+}
+
+<===>
+================================================================================
+<===> infinity/input.scss
+@use "sass:math" as math;
+a {b: math.log(1 / 0)}
+
+<===> infinity/output.css
+a {
+  b: Infinity;
+}
+
+<===>
+================================================================================
+<===> base/input.scss
+@use "sass:math" as math;
+a {b: math.log(2, 10)}
+
+<===> base/output.css
+a {
+  b: 0.3010299957;
+}
+
+<===>
+================================================================================
+<===> base/negative/input.scss
+@use "sass:math" as math;
+a {b: math.log(2, -1)}
+
+<===> base/negative/output.css
+a {
+  b: NaN;
+}
+
+<===>
+================================================================================
+<===> base/zero/input.scss
+@use "sass:math" as math;
+a {b: math.log(2, 0)}
+
+<===> base/zero/output.css
+a {
+  b: 0;
+}
+
+<===>
+================================================================================
+<===> base/one/input.scss
+@use "sass:math" as math;
+a {b: math.log(2, 1)}
+
+<===> base/one/output.css
+a {
+  b: Infinity;
+}
+
+<===>
+================================================================================
+<===> error/number_has_units/input.scss
```

Comment:
Don't forget to test type errors as well.

Reply:
Done.